customers/models.py

```diff
@@ -21,6 +21,11 @@ class CustomerProfile(TimeStampedModel, UUIDModel):
     )
     comment = models.TextField(verbose_name=_("comment"), blank=True)
 
+    class Meta:
+        verbose_name = _("customer profile")
+        verbose_name_plural = _("customer profiles")
+        ordering = ("id",)
```

Comment:
Does it make sense ordering by `id` if it's a UUID?

Reply:
It has absolutely no effect 😅 `"id"` is default field for ordering anyway, just wanted to make it explicit.

Reply:
no other field could fit better for odering at this point

Reply:
Alright :) just curious if there was any other expected result. 👍